release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3315,3 +3315,51 @@ You can view the container images in this release by running the following comma
+
+* Previously, a bug in systemd might cause the `coreos-multipath-trigger.service` unit to hang forever. As a result the system would never finish booting. With this release, the systemd unit was removed and the issue is fixed. (link:https://issues.redhat.com/browse/OCPBUGS-35748[*OCPBUGS-35748*])
+
+* Previously, the KMS key was applied as an empty string. This always applied a KMS key that was invalid. With this release, the empty string is removed and the KMS key is only applied when one exists from the `install-config.yaml`. (link:https://issues.redhat.com/browse/OCPBUGS-35531[*OCPBUGS-35531*])
+
+* Previously, there was no validation of the values for confidential compute and on host maintenance set by the user. With this release, when confidential compute is enabled by the user the value for `onHostMaintenance` must be set to `onHostMaintenance: Terminate`. (link:https://issues.redhat.com/browse/OCPBUGS-35493[*OCPBUGS-35493*])
+
+* Previously, in User Provisioned Infrastructure (UPI) or clusters that were upgraded from older versions, `failureDomains` may be missing in Infrastructure objects which caused certain checks to fail. With this release, a fallback `failureDomains` is synthesized from `cloudConfig` if none are available in `infrastructures.config.openshift.io`. (link:https://issues.redhat.com/browse/OCPBUGS-35446[*OCPBUGS-35446*])
+
+* Previously, when a new version of a Custom Resource Definition (CRD) specified a new conversion strategy, this conversion strategy was expected to successfully convert resources. This was not the case because Operator Lifecycle Manager (OLM) cannot run the new conversion strategies for CRD validation without actually performing the update operation. With this release, the OLM implementation generates a warning message during the update process when CRD validations fail with the existing conversion strategy and the new conversion strategy is specified in the new version of the CRD. (link:https://issues.redhat.com/browse/OCPBUGS-35373[*OCPBUGS-35373*])
+
+* Previously, Amazon Web Services (AWS) HyperShift clusters leveraged their Amazon Virtual Private Cloud (VPC)'s primary classless inter-domain routing (CIDR) range to generate security group rules on the data plane. As a consequence, installing AWS HyperShift clusters into an AWS VPC with multiple CIDR ranges could cause the generated security group rules to be insufficient. With this update, security group rules are generated based on the provided Machine CIDR range to resolve this issue. (link:https://issues.redhat.com/browse/OCPBUGS-35056[*OCPBUGS-35056*])
+
+* Previously, a change on what the Azure API returns for a subnet would cause the Installer to crash. With this release, the code can handle both the old and new data for subnets as well as returning an error in case the expected information is not found. (link:https://issues.redhat.com/browse/OCPBUGS-35023[*OCPBUGS-35023*])
```

Comment:
```suggestion
* Previously, a change to the field that Azure API returns for a subnet would cause the Installer to crash. With this release, the code can handle both the old and new data for subnets as well as returning an error in case the expected information is not found. (link:https://issues.redhat.com/browse/OCPBUGS-35023[*OCPBUGS-35023*])
```